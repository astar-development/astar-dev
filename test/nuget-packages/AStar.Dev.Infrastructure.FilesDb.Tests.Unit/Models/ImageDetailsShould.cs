using AStar.Dev.Utilities;
using JetBrains.Annotations;

namespace AStar.Dev.Infrastructure.FilesDb.Models;

[TestSubject(typeof(ImageDetails))]
public class ImageDetailsShould
{
    [Fact]
    public void ContainTheExpectedProperties()
        => new ImageDetails { Id = 1, FileDetailsId = 2, Width = 123, Height = 456 }.ToJson().ShouldMatchApproved();

    [Fact]
    public void ReturnTheExpectedToString()
        => new ImageDetails { Id = 1, FileDetailsId = 2, Width = 123, Height = 456 }.ToString().ShouldMatchApproved();
}
