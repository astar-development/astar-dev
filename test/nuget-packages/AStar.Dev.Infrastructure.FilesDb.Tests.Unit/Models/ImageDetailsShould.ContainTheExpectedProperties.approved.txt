{
  "id": 1,
  "width": 123,
  "height": 456,
  "fileDetailsId": 2
}